src/zenml/integrations/__init__.py

```diff
@@ -32,6 +32,7 @@
 from zenml.integrations.pytorch_lightning import (  # noqa
     PytorchLightningIntegration,
 )
+from zenml.integrations.scipy import ScipyIntegration
```

Comment:
```suggestion
from zenml.integrations.scipy import ScipyIntegration  # noqa
```

I assume we'll want to skip linting here too.